zyte_common_items/pages.py

```diff
@@ -410,6 +410,7 @@ class Processors(Page.Processors):
         breadcrumbs = [breadcrumbs_processor]
         description = [description_processor]
         descriptionHtml = [description_html_processor]
+        gtin = [gtin_processor]
```

Comment:
Missed this in the original PR. I wonder if we can deduplicate BaseFooPage.Processors and FooPage.Processors.